db/migrate/20170312180419_add_userid_to_answers.rb

```diff
@@ -0,0 +1,5 @@
+class AddUseridToAnswers < ActiveRecord::Migration[5.0]
+  def change
+    add_column :answers, :user_id, :integer
+  end
+end
```

Comment:
Again, I would expect there to be more changes to the database due to adding a couple `has_many` relationships.